src/game_inventory.cpp

```diff
@@ -641,11 +641,11 @@ class read_inventory_preset: public pickup_inventory_preset
                 return unlearned > 0 ? to_string( unlearned ) : std::string();
             }, _( "RECIPES" ), unknown );
 
-            append_cell( [ this ]( const item_location & loc ) -> std::string {
+            append_cell( [ p, this ]( const item_location & loc ) -> std::string {
```

Comment:
This needlessly copies `p` (`player` is expensive to copy). Should be `&p`.

Reply:
I'll fix this as soon as I can. Thanks for telling me.